events_worker.go

```diff
@@ -178,10 +191,13 @@ func (w *EventsWorker) newProcessEventFunc(handlers map[proto.Message][]EventHan
 			log.Info("Event processed successfully")
 		}
 
-		// For now, we commit the message even if the handler failed to process it.
-		//
-		// TODO: add a configuration option to allow the user to choose whether to commit the message or not.
-		// Or maybe even publish the message to a dead-letter topic?
+		if handleErr != nil && errorMode == ShutdownOnError {
+			w.Logger.WithField("event", event.ProtoName).Errorf("Cannot process event: %v", handleErr)
+			w.cancelFunc()
+			return handleErr
+		}
+		// Else: errorMode == Skip
```

Comment:
This should be a `// TODO: ` comment, not a pseudo code

Reply:
TODO is placed on ErrorHandlingStrategy defenition. For now: just deleted the pseudo code.